packages/js/src/settings/routes/templates/taxonomy.js

```diff
@@ -231,27 +233,31 @@ const Taxonomy = ( { name, label, postTypes: postTypeNames } ) => {
 							/>
 						</FeatureUpsell>
 					</FieldsetLayout>
-					<hr className="yst-my-8" />
-					<FieldsetLayout
-						title={ __( "Additional settings", "wordpress-seo" ) }
-					>
-						<FormikValueChangeField
-							as={ ToggleField }
-							type="checkbox"
-							name={ `wpseo_titles.display-metabox-tax-${ name }` }
-							id={ `input-wpseo_titles-display-metabox-tax-${ name }` }
-							label={ __( "Enable SEO controls and assessments", "wordpress-seo" ) }
-							description={ __( "Show or hide our tools and controls in the content editor.", "wordpress-seo" ) }
-							className="yst-max-w-sm"
-						/>
-						{ name === "category" && <FormikFlippedToggleField
-							name="wpseo_titles.stripcategorybase"
-							id="input-wpseo_titles-stripcategorybase"
-							label={ __( "Show the categories prefix in the slug", "wordpress-seo" ) }
-							description={ stripCategoryBaseDescription }
-							className="yst-max-w-sm"
-						/> }
-					</FieldsetLayout>
+
+					{ showUI &&
+					<>
+						<hr className="yst-my-8" />
+						<FieldsetLayout
+							title={ __( "Additional settings", "wordpress-seo" ) }
+						>
+							<FormikValueChangeField
+								as={ ToggleField }
+								type="checkbox"
+								name={ `wpseo_titles.display-metabox-tax-${ name }` }
+								id={ `input-wpseo_titles-display-metabox-tax-${ name }` }
+								label={ __( "Enable SEO controls and assessments", "wordpress-seo" ) }
+								description={ __( "Show or hide our tools and controls in the content editor.", "wordpress-seo" ) }
+								className="yst-max-w-sm"
+							/>
+							{ name === "category" && <FormikFlippedToggleField
+								name="wpseo_titles.stripcategorybase"
+								id="input-wpseo_titles-stripcategorybase"
+								label={ __( "Show the categories prefix in the slug", "wordpress-seo" ) }
+								description={ stripCategoryBaseDescription }
+								className="yst-max-w-sm"
+							/> }
```

Comment:
I think we want to keep this settings regardless of the show UI

Reply:
But then we are left with empty additional settings...
<img width="1116" alt="image" src="https://user-images.githubusercontent.com/65466507/215790626-cb0f12c4-afd9-494f-afc2-73d1c66addd3.png">